File: DesignerPro/TableOfContents/C#/TableOfContents/ExportViewerFactory.cs
using GrapeCity.ActiveReports.Design.Advanced;
using GrapeCity.ActiveReports.Viewer.Win.Internal.Export;
using GrapeCity.ActiveReports.Win.Export;

namespace GrapeCity.ActiveReports.Samples.TableOfContents
{
	internal class ExportViewerFactory : IExportViewerFactory
	{
		public IExportViewer CreateExportViewer(Viewer.Win.Viewer viewer)
		{
			return new ExportViewer(viewer);
		}
	}
}

File: Advanced/PageAndRDL/CustomDataProvider/VB.NET/TestDesignerPro/ExportViewerFactory.vb
Imports GrapeCity.ActiveReports.Design.Advanced
Imports GrapeCity.ActiveReports.Viewer.Win.Internal.Export

Friend Class ExportViewerFactory
	Implements IExportViewerFactory
	Public Function CreateExportViewer(viewer As Viewer.Win.Viewer) As IExportViewer Implements IExportViewerFactory.CreateExportViewer
		Return New ExportViewer(viewer)
	End Function
End Class
